avaframe/runTestFP.py

```diff
@@ -0,0 +1,109 @@
+"""
+    Run script for running python DFA kernel
+"""
+import os
+import time
+import numpy as np
+
+# Local imports
+import avaframe.in3Utils.initializeProject as initProj
+import avaframe.in3Utils.fileHandlerUtils as fU
+from avaframe.in1Data import getInput as gI
+import avaframe.com1DFAPy.com1DFA as com1DFA
+from avaframe.com1DFAPy import runCom1DFA
+import avaframe.ana1Tests.FPtest as FPtest
+
+# from avaframe.DFAkernel.setParam import *
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import avaframe.out3Plot.makePalette as makePalette
+
+# +++++++++SETUP CONFIGURATION++++++++++++++++++++++++
+# log file name; leave empty to use default runLog.log
+logName = 'runFlatPlaneTest'
+
+# Load avalanche directory from general configuration file
+cfgMain = cfgUtils.getGeneralConfig()
+avalancheDir = 'data/avaFPtest'
+# set module name, reqiured as long we are in dev phase
+# - because need to create e.g. Output folder for com1DFAPy to distinguish from
+# current com1DFA
+modName = 'com1DFAPy'
+
+# Clean input directory(ies) of old work and output files
+initProj.cleanSingleAvaDir(avalancheDir, keep=logName)
+
+# Start logging
+log = logUtils.initiateLogger(avalancheDir, logName)
+log.info('MAIN SCRIPT')
+log.info('Current avalanche: %s', avalancheDir)
+
+# Load configuration
+FPCfg = os.path.join(avalancheDir, 'Inputs', 'FlatPlane_com1DFACfg.ini')
+cfg = cfgUtils.getModuleConfig(com1DFA, FPCfg)
+cfgGen = cfg['GENERAL']
+cfgFP = cfg['FPSOL']
+flagDev = cfg['FLAGS'].getboolean('flagDev')
```

Comment:
not required?

Reply:
however it can stay one wants to change it in the ini file, true

Reply:
I use them I think